src/main/java/uk/gov/hmcts/reform/sscs/ccd/service/SearchCcdCaseService.java

```diff
@@ -87,6 +87,9 @@ private SscsCaseDetails findCaseByCaseRefOrCaseIdRetryLogic(SscsCaseData caseDat
         SscsCaseDetails sscsCaseDetails = null;
         if (StringUtils.isNotBlank(caseData.getCaseReference())) {
             sscsCaseDetails = this.findCaseByCaseRef(caseData.getCaseReference(), idamTokens);
```

Comment:
why make this call if we're making another call just below to retrieve caseDetails

Reply:
The second call retrieves the case details from ccd as opposed to elastic search (we still need to make the first call to get the case from case ref)